model/src/main/java/org/cbioportal/model/CancerStudy.java

```diff
@@ -2,13 +2,101 @@
 
 import org.cbioportal.model.summary.CancerStudySummary;
 
-import java.io.Serializable;
-import java.util.Date;
 
 public class CancerStudy extends CancerStudySummary {
 
     private TypeOfCancer typeOfCancer;
     private Integer sampleCount;
+    private String stableID;
+    private Integer allCount;
```

Comment:
allCount is the same thing with sampleCount, you can delete allCount and use sampleCount, because the name is more clear, allCount isn't self explaining, all count of what?